src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -152,6 +153,14 @@ public static function register($handler = null, $replace = true)
         return $handler;
     }
 
+    public function __construct(BufferingLogger $bootstrappingLogger = null)
+    {
+        if ($bootstrappingLogger) {
```

Comment:
I don't think so: it's a BufferingLogger that is used for the bootstrapping phase, thus a bootstrappingLogger insternally